### PR TITLE
FIX: Default language overwritten on thirdparty edit/update

### PR DESCRIPTION
**Pull request by Tim-Otte**
Fixed an error where the selected default language would be ignored in the UI and overwritten on an update

### CODE REVIEW

File: htdocs/core/class/html.formadmin.class.php [hunk 1]
```diff
@@ -125,6 +125,7 @@ public function select_language($selected = '', $htmlname = 'lang_id', $showauto
 		{
 			include_once DOL_DOCUMENT_ROOT.'/core/lib/ajax.lib.php';
 			$out .= ajax_combobox($htmlname);
```
Comment (eldy) [inline htdocs/core/class/html.formadmin.class.php:127]: ajax_combobox() is a generic component that should works with all combolist.
Can you describe how to reproduce the bug ?

Comment (Tim-Otte) [reply htdocs/core/class/html.formadmin.class.php:127]: Create a thirdparty and set its default language to any value that is not your default language. Then click on modify in the same thirdparty and save without changing anything. The default language now should be something else than you selected. In the standard html select, the correct value is set but in the select2 input the value is different.

Comment (Tim-Otte) [reply htdocs/core/class/html.formadmin.class.php:127]: I had this bug with system wide default language 'de_DE' and the thirdparty default language 'en_GB'

Comment (eldy) [reply htdocs/core/class/html.formadmin.class.php:127]: Problem should come from somewhere else.
I can't reproduce with 12.0.3. Did you ?

Comment (Tim-Otte) [reply htdocs/core/class/html.formadmin.class.php:127]: I have this error in 12.0.2. I currently have no capacities to test with 12.0.3 but will try it as soon as possible

**Comment by eldy** [comment]
I have some reason to think this may be fixed or no more applicable into more recent version. Does this bug still occurs with last stable version ?
If yes, please reopen the ticket and mention the recent version you tried with...

**Pull request closed without merging**